src/bloom/domain/spire_ais_data.py

```diff
@@ -41,7 +41,7 @@ class SpireAisData(BaseModel):
     voyage_update_timestamp: datetime | None
     created_at: Union[datetime, None] = None
 
-    def map_from_spire(spire_update_timestamp: datetime, vessel: dict[str, Any]) -> Self:
+    def map_from_spire(spire_update_timestamp: datetime, vessel: dict[str, Any]):
```

Comment:
Le fait de supprimer -> Self génère une erreur Ruff
 ANN201 Missing return type annotation for public function `map_from_spire`
Est-ce que Self est le bon type de retour ? si oui il faut maintenir l'import de Self

Reply:
on tourne pas en rond là ... ?
`target-version = "py310"` -> ruff me semble configuré pour valider du python 3.10 et d'un autre côté on build en 3.8 et 3.9.

Reply:
résolu avec un `#noqa`